Client/Frontend/Reader/ReaderModeStyleViewController.swift

```diff
@@ -5,78 +5,123 @@
 import Foundation
 import UIKit
 
+private struct ReaderModeStyleViewControllerUX {
+    static let RowHeight = 50
+
+    static let Width = 270
+    static let Height = 4 * RowHeight
+
+    static let FontTypeRowBackground = UIColor(rgb: 0xfbfbfb)
+    static let FontTypeTitleFontSansSerif = UIFont(name: "FiraSans", size: 16)
+    static let FontTypeTitleFontSerif = UIFont(name: "Charis SIL", size: 16)
+
+    static let FontTypeTitleSelectedColor = UIColor(rgb: 0x333333)
+    static let FontTypeTitleNormalColor = UIColor.lightGrayColor() // TODO THis needs to be 44% of 0x333333
+
+    static let FontSizeRowBackground = UIColor(rgb: 0xf4f4f4)
+    static let FontSizeLabelFontSansSerif = UIFont(name: "FiraSans-Book", size: 22)
+    static let FontSizeLabelFontSerif = UIFont(name: "Charis SIL", size: 22)
+    static let FontSizeLabelColor = UIColor(rgb: 0x333333)
+    static let FontSizeButtonTextColorEnabled = UIColor(rgb: 0x333333)
+    static let FontSizeButtonTextColorDisabled = UIColor.lightGrayColor() // TODO THis needs to be 44% of 0x333333
+    static let FontSizeButtonTextFont = UIFont(name: "FiraSans-Book", size: 22)
+
+    static let ThemeRowBackgroundColor = UIColor.whiteColor()
+    static let ThemeTitleFontSansSerif = UIFont(name: "FiraSans-Light", size: 15)
+    static let ThemeTitleFontSerif = UIFont(name: "Charis SIL", size: 15)
```

Comment:
The font differences here are a bit strange. You should probably just use 15 or 16 for both this and the FontTypeTitle's. I'd like to use things from AppConstants here, but I know this is a bit specialized I guess. Watch for them appearing again and move them then?